netmap/types.proto

```diff
@@ -109,11 +109,13 @@ message Replica {
 // storage policy definition languages.
 message PlacementPolicy {
   // Rules to set number of object replicas and place each one into a named
-  // bucket
+  // bucket. Limited to 256 items.
   repeated Replica replicas = 1 [json_name = "replicas"];
 
-  // Container backup factor controls how deep NeoFS will search for nodes
-  // alternatives to include into container's nodes subset
+  // Container backup factor (CBF) controls how deep NeoFS will search for
+  // alternative nodes to include into container's nodes subset. In total,
+  // the number of container nodes is Replica count times CBF. This number
```

Comment:
`Selector`, not `Replica`:
* https://github.com/nspcc-dev/neofs-sdk-go/blob/5f693d22cfe6350060e546e383c293317253352a/netmap/selector.go#L36-L44
* https://github.com/nspcc-dev/neofs-sdk-go/blob/5f693d22cfe6350060e546e383c293317253352a/netmap/selector.go#L81

Reply:
Let's try this way.